jamjar/jamjar/videos/serializers.py

```diff
@@ -1,10 +1,13 @@
 from rest_framework import serializers
 from jamjar.videos.models import Video, Edge
-from jamjar.concerts.serializers import ConcertSerializer
+from jamjar.artists.serializers import ArtistSerializer
+from jamjar.users.serializers import UserSerializer
 
 import os
 
 class VideoSerializer(serializers.ModelSerializer):
+    artists = ArtistSerializer(many=True)
```

Comment:
I think I have a solution to this though.  Will work on it in a minute, helping jess get setup with stuff rn